src/run_coloc.py

```diff
@@ -30,14 +31,17 @@ def main(cfg: DictConfig) -> None:
 
     # Load data
     credible_sets = (
-        etl.spark.read.parquet(cfg.etl.coloc.inputs.credible_set)
-        # .filter(f.col("chrom") == "22")  # for debugging
-        .withColumn(
-            "studyKey",
-            f.xxhash64(*["type", "study_id", "phenotype_id", "bio_feature"]),
+        _extract_credible_sets(
+            etl.spark.read.parquet(cfg.etl.coloc.inputs.study_locus_idx)
         )
-        # Exclude studies without logABFs available
         .filter(f.col("logABF").isNotNull())
+        .filter(f.col("chromosome") == "22")  # for testing
```

Comment:
We need to run it without this filter. Make sure this PR is not causing any unintended shuffling. Everything (excluding sumstats metadata) was taking ~25 min in the coloc machine

Reply:
The whole dataset is here `gs://genetics_etl_python_playground/XX.XX/output/python_etl/parquet/coloc_refactor_all`
[Job](https://console.cloud.google.com/dataproc/jobs/53fab164febb4e149b6a0cf5a1adee8c?region=europe-west1&project=open-targets-genetics-dev) took 18 mins.